### PR TITLE
A few small fixes and minor improvements

### PR DESCRIPTION
**Pull request by marcelveldt**
- extend the heartbeat interval from 10 seconds to 120 seconds to reduce network traffic
- disable the fabric filter on the read/subscription request, this will allow managing fabrics*

* this will also bring the possibility to actually remove a device from our fabric

### CODE REVIEW

File: scripts/beautify_diagnostics.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 """Script to beautify diagnostics output."""
 import json
+from os.path import abspath, dirname
 import sys
 
 from chip.clusters.ClusterObjects import ALL_ATTRIBUTES, ALL_CLUSTERS
 import yaml
 
-from matter_server.client.models.device_types import ALL_TYPES
+sys.path.insert(1, dirname(dirname(abspath(__file__))))
```
Comment (MartinHjelmare) [inline scripts/beautify_diagnostics.py:9]: Why is this needed?

Comment (marcelveldt) [reply scripts/beautify_diagnostics.py:9]: so you can run the script without the lib installed, just from your project root

Comment (marcelveldt) [reply scripts/beautify_diagnostics.py:9]: Meh, I don't want to on my dev machine

Comment (MartinHjelmare) [reply scripts/beautify_diagnostics.py:9]: Install with:

```
pip install -e .[server]
```

That will symlink the folder.

Comment (marcelveldt) [reply scripts/beautify_diagnostics.py:9]: Unfortunately that does not work on MacOS. It actually works when sqitching from zsh shell to bash but then the next issue is the wheels requirement which can not be met so for now we'll have to leave the workaround in place

**Review by MartinHjelmare — APPROVED** [review]

File: matter_server/client/models/node.py [hunk 2]
```diff
@@ -279,7 +279,7 @@ def get_compose_parent(self, endpoint_id: int) -> MatterEndpoint | None:
         return None
 
     def get_compose_child_ids(self, endpoint_id: int) -> tuple[int, ...] | None:
-        """Return endpoint ID's of any childs if the endpoint represents a Composed device."""
+        """Return endpoint ID's of any child's if the endpoint represents a Composed device."""
```
Comment (MartinHjelmare) [inline matter_server/client/models/node.py:282]:
```suggestion
        """Return endpoint IDs of any child if the endpoint represents a Composed device."""
```

File: scripts/beautify_diagnostics.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from chip.clusters.ClusterObjects import ALL_ATTRIBUTES, ALL_CLUSTERS
 import yaml
 
-from matter_server.client.models.device_types import ALL_TYPES
+from matter_server.client.models.device_types import ALL_TYPES  # noqa: E402
```
Comment (MartinHjelmare) [inline scripts/beautify_diagnostics.py:8]: Is the noqa needed?

Comment (marcelveldt) [reply scripts/beautify_diagnostics.py:8]: nope, leftover
